music_assistant/server/providers/bluesound/__init__.py

```diff
@@ -0,0 +1,586 @@
+        """Send VOLUME MUTE command to given player."""
+        if bluos_player := self.bluos_players[player_id]:
+            await bluos_player.client.volume(mute=muted)
+            # Optimistic state, reduces interface lag
+            mass_player = self.mass.players.get(self.player_id)
+            mass_player.volume_mute = muted
+
+    async def cmd_seek(self, player_id: str, position: int) -> None:
+        """Handle SEEK command for given queue.
+
+        - player_id: player_id of the player to handle the command.
+        - position: position in seconds to seek to in the current playing item.
+        """
+        if bluos_player := self.bluos_players[player_id]:
+            await bluos_player.client.play(seek=position)
+
+        # OPTIONAL - required only if you specified PlayerFeature.SEEK
+        # this method should handle the seek command for the given player.
+        # the position is the position in seconds to seek to in the current playing item.
+
+    async def play_media(
+        self, player_id: str, media: PlayerMedia, timeout: float | None = None
+    ) -> None:
+        """Handle PLAY MEDIA on given player using the provided URL."""
+        # if status_result.state:
+        #     # this should be already handled by the player manager, but just in case...
+        #     msg = (
+        #         f"Player {mass_player.display_name} cannot "
+        #         "accept play_media command, it is synced to another player."
+        #     )
+        #     raise PlayerCommandFailed(msg)
+
+        # Prepare the URL to play
+        # media_url = f"{self.mass.streams.base_url}/play/{media.queue_id}"
+
+        # Use the play_url method to start playback
+        if bluos_player := self.bluos_players[player_id]:
+            playback_state = await bluos_player.client.play_url(media.uri, timeout=timeout)
+
+        # Optionally, handle the playback_state or additional logic here
+        if playback_state != "playing":
+            raise PlayerCommandFailed("Failed to start playback.")
+
+    async def enqueue_next_media(self, player_id: str, media: PlayerMedia) -> None:
+        """
+        Handle enqueuing of the next (queue) item on the player.
+
+        Only called if the player supports PlayerFeature.ENQUE_NEXT.
+        Called about 1 second after a new track started playing.
+        Called about 15 seconds before the end of the current track.
+
+        A PlayerProvider implementation is in itself responsible for handling this
+        so that the queue items keep playing until its empty or the player stopped.
+
+        This will NOT be called if the end of the queue is reached (and repeat disabled).
+        This will NOT be called if the player is using flow mode to playback the queue.
+        """
+        # OPTIONAL - required only if you specified PlayerFeature.ENQUEUE_NEXT
+        # this method should handle the enqueuing of the next queue item on the player.
```

Comment:
```suggestion
```

Bluesound (unfortunately) doesn't support enqueuing, at least not in their public api, so this can be removed